### PR TITLE
Fixed hanging quotemark in moderate difficulty text

### PR DESCRIPTION
**Pull request by MartinLyne**

### CODE REVIEW

File: GameData/BioMass/Plugin/pluginData/BioMassDifficulty/Moderate.txt [hunk 1]
```diff
@@ -176,7 +176,7 @@ DifficultySetting{
 @PART[BioMass_KoylentMachine]{
    @category=Utility
    @title=Koylent Maker
-   @description=When you are low on food and far from home things can look bleak. To stave off sure depression, DG Aero-Space has designed the Koylent Maker to supply all your food-on-the-go needs. Various vacuums and pistons combine biocake and seeds into a gastronomic delight DG Aero-Space calls "Koylent, though most just call it "food."
+   @description=When you are low on food and far from home things can look bleak. To stave off sure depression, DG Aero-Space has designed the Koylent Maker to supply all your food-on-the-go needs. Various vacuums and pistons combine biocake and seeds into a gastronomic delight DG Aero-Space calls "Koylent", though most just call it "food."
```
Comment (TJ09) [inline GameData/BioMass/Plugin/pluginData/BioMassDifficulty/Moderate.txt]: FWIW, proper US English is to put the comma or period inside the quotation marks.

Comment (MartinLyne) [inline GameData/BioMass/Plugin/pluginData/BioMassDifficulty/Moderate.txt]: Jnot sure I understand, my addition was a matching quote after the word Koylent. Before there were only 3 quotes. I left the ending quote in US-en

Comment (TJ09) [inline GameData/BioMass/Plugin/pluginData/BioMassDifficulty/Moderate.txt]: ``"Koylent,"``, rather than ``"Koylent",``

**Comment by seanth** [comment]
In this case, I prefer the UK-en style of the comma outside the end quote for "Koylent".

**Pull request merged**